core/station/impl/Cargo.toml

```diff
@@ -41,6 +41,17 @@ thiserror = { workspace = true }
 uuid = { workspace = true, features = ['v4'] }
 station-api = { path = '../api', version = '0.0.2-alpha.3' }
 upgrader-api = { path = '../../upgrader/api', version = '0.0.2-alpha.3' }
+alloy = { version = "0.2.0", features = [
+    "signers",
+    "k256",
+    "consensus",
+    "eips",
+] }
+alloy-chains = "0.1.23"
+evm-rpc-canister-types = "1.0.0"
+maplit = "1.0.2"
+serde_json = { workspace = true }
+rand = "0.8.5"
```

Comment:
random numbers in a canister should retrieved with the `raw_rand` call, ideally the `orbit-essentials` crate could be extended with `pub fn random_number()` that there it would use [this rand](https://github.com/dfinity/orbit/blob/ff18d25c56c8bbcd6c8e448029a3b4157bd3faa7/libs/orbit-essentials/src/utils/random.rs#L9).